.github/workflows/test.yaml

```diff
@@ -0,0 +1,65 @@
+name: Test
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+      max-parallel: 1
+
+    steps:
+      - name: Print Github Context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo $GITHUB_CONTEXT
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: pycache
+        uses: actions/cache@v2
+        with:
+          path: .venv
+          key: python-${{ github.sha }}
```

Comment:
Is this working fine? It seems nobody uses `.venv`. Is it automatically used? Do you intend to cache `.cache/pip` instead?
https://github.com/actions/cache/blob/main/examples.md#python---pip

Reply:
Ooops, I did miss it.